apps/package.json

```diff
@@ -127,7 +127,7 @@
     "radium": "0.18.1",
     "react": "15.2.1",
     "react-addons-test-utils": "15.2.1",
-    "react-bootstrap": "0.29.5",
+    "react-bootstrap": "^0.30.1",
```

Comment:
should we tie to exactly "0.30.1" so that we dont have to worry about different engineers ending up on slightly different versions?